src/mantine-dates/src/components/TimeInputBase/AmPmInput/AmPmInput.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useRef, forwardRef } from 'react';
+import { useMergedRef } from '@mantine/hooks';
+import { MantineSize } from '@mantine/core';
+import useStyles from '../TimeInputBase.styles';
+
+interface AmPmSelectProps
+  extends Omit<React.ComponentPropsWithoutRef<'input'>, 'onChange' | 'size'> {
+  /** Called with onChange event */
+  onChange(value: string, triggerShift: boolean): void;
+
+  /** Called when input loses focus, used to format value */
+  setValue(value: string): void;
+
+  /** Colon text size */
+  size?: MantineSize;
+
+  /** Displays error message after input */
+  error?: React.ReactNode;
```

Comment:
Why would you need error prop? It is not used anywhere

Reply:
Your right that was a old prop (I forgot to remove it)